src/modules/populateList.js

```diff
@@ -1,45 +1,8 @@
 import vertBtnSvg from '../icons/vert.svg';
 import { textAreaChangeHandler } from './editTask';
+import { checkBoxHandler, liCheckedToggle } from './checkBoxControl';
 
-const listContainer = document.querySelector('#list-container');
-const ul = listContainer.querySelector('ul');
-
-function liCheckedToggle(checkBox) {
-  checkBox.parentElement.classList.toggle('checked');
-}
-
-function updateLocalStorage(completeStatus, checkBox) {
-  // find task location
-  const checkBoxes = ul.querySelectorAll('li > input[type=checkbox]');
-  let index = 0;
-  checkBoxes.forEach((el, i) => {
-    if (el === checkBox) {
-      index = i;
-    }
-  });
-  // find task within localstorage
-  const tasks = JSON.parse(localStorage.getItem('tasks')) || [];
-  tasks.forEach((task) => {
-    console.log(index);
-    if (task.index === index) {
-      task.completed = completeStatus;
-      console.log(task);
```

Comment:
Please ensure that you do not have console.log(task); in your codebase, it makes your code clean and for security reasons.